backstop/tests/table.html

```diff
@@ -273,15 +273,66 @@ <h2>Default</h2>
             </div>
             <div class="iui-cell iui-slot">
               <button
-                class="iui-button iui-borderless"
+                class="iui-button iui-borderless iui-dropdown"
```

Comment:
You don't need this `iui-dropdown` class, it's only used for adjusting the spacing when the caret icon is used.